src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitParameters.java

```diff
@@ -95,15 +97,20 @@ public boolean filter(PeakListRow row) {
       "Filter rows", "Limit the exported rows to those with MS/MS data or annotated rows",
       RowFilter.values(), RowFilter.ONLY_WITH_MS2);
 
+
   // public static final BooleanParameter OPEN_GNPS = new BooleanParameter("Open GNPS website",
   // "Opens the super quick start of GNPS feature based networking in the standard browser.",
   // false);
 
   public static final BooleanParameter OPEN_FOLDER =
       new BooleanParameter("Open folder", "Opens the export folder", false);
 
+
+  public static final OptionalModuleParameter<MsMsSpectraMergeParameters> MERGE_PARAMETER = new OptionalModuleParameter<>("Merge MS/MS", "Merge high qualitative MS/MS instead of exporting just the most intensive one.", new MsMsSpectraMergeParameters(), true);
```

Comment:
Also a minor thing, but the correct term is "high-quality", not "high qualitative"